server/odc-service/src/main/java/com/oceanbase/odc/service/task/runtime/PreCheckTaskParameters.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.task.runtime;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Set;
+
+import com.oceanbase.odc.core.shared.constant.TaskType;
+import com.oceanbase.odc.service.collaboration.environment.model.Environment;
+import com.oceanbase.odc.service.connection.model.ConnectionConfig;
+import com.oceanbase.odc.service.objectstorage.model.ObjectMetadata;
+import com.oceanbase.odc.service.regulation.risklevel.model.RiskLevelDescriber;
+import com.oceanbase.odc.service.regulation.ruleset.model.Rule;
+
+import lombok.Data;
+
+/**
+ * @author gaoda.xy
+ * @date 2024/1/30 11:06
+ */
+@Data
+public class PreCheckTaskParameters implements Serializable {
```

Comment:
implements Serializable is unnecessary

Reply:
This is done to be consistent with the style of the existing task parameters, and shouldn't introduce any additional disadvantages.